File: C Programming/Assignment/Assignment 7/reverse.c
#include<stdio.h>
void main()
{
	int arr[5],i;
	printf("Enter a element of array");
	for(i=0;i<5;i++)
	{
		scanf("%d",&arr[i]);
	}
	printf("Reverse of array");
	for(i=4;i>=0;i--)
	{
		printf("%d ",arr[i]);
	}
}

File: Core Java/Assignment/16 Sep Array/Reverse.java
import java.util.Scanner;
class ReverseArray
{
  public static void main(String args[])
 {
   Scanner sc=new Scanner(System.in);
   int arr[]=new int[5];
   int i;
   System.out.println("Enter the element of array");
   for(i=0;i<arr.length;i++)
   {
    arr[i]=sc.nextInt();
  }
  
   System.out.println("Reverse element of array:");
   for(i=arr.length-1;i>=0;i--)
   {
     System.out.print(arr[i]+" ");
  }
}
}
   